### PR TITLE
Added questions 12,13,14. 

### PR DESCRIPTION
**Pull request by j4sum**
## Type Of Change
- [ ] Update documentation
- [x] Add feature
- [ ] Fix 

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] Commit message is as per guidelines in the [contributor guide](https://github.com/WomenWhoCode/london/blob/main/CONTRIBUTING.md)

### CODE REVIEW

**Comment by dricazenck** [comment]
Please always add screenshots.

When you change CSS also please add 2 screenshots before and after, please.

File: _data/faq.yml [hunk 1]
```diff
@@ -52,3 +52,35 @@
...
+    1. Professional development: To provide mentees with guidance and support as they navigate their career and personal growth. <br>
+    2. Skill development: To help mentees develop specific skills and knowledge that will be valuable to them in their current or future roles. <br>
+    3. Career guidance: To help mentees set and achieve career goals, and to provide them with the tools and resources needed to succeed in their chosen field. <br>
+    4. Personal development: To provide mentees with guidance and support as they navigate their personal growth and well-being. <br> <br>
+    As a mentor, you will be expected to provide guidance, advice, and support to your mentee as they navigate their professional and personal development. You will be expected to share your own experiences and knowledge, and provide constructive feedback. You will also be expected to help the mentee set and achieve goals, and to introduce them to new opportunities that can help them advance in their career. <br>
+    Please, find more information in Code of Conduct for Mentors.
```
Comment (dricazenck) [inline _data/faq.yml:66]:
```suggestion
    Please, find more information in <a href="https://womenwhocode.github.io/london/code-of-conduct)">Code of Conduct for Mentors</a>.
```

I think that was what @nora-weisser suggested

File: _data/faq.yml [hunk 2]
```diff
@@ -52,3 +52,35 @@
...
+  link: https://womenwhocode.github.io/london/code-of-conduct
```
Comment (dricazenck) [inline _data/faq.yml:67]: link attribute can be removed.

File: _data/faq.yml [hunk 3]
```diff
@@ -52,3 +52,35 @@
...
+  index: 13
+
+- question: What is the time commitment required for the mentorship program? How often will I be expected to meet with the mentee, and how long the cycle last? 
+  answer: |
+    
+    1. Meeting frequency: Mentors and mentees are expected to meet on a regular basis, such as weekly or monthly. It should be discussed between the mentor and mentee during the introduction meeting. <br>
+    2. Meeting duration: Meetings can be scheduled for different durations, usually lasting between 30 minutes to an hour depending on availability and needs. <br>
+    3. Programme duration: Mentorship programs lasts from 20th of April till 1st of November 2023. <br><br>
+    It's important for both the mentor and mentee to have a clear understanding of the time commitment required for the mentorship programme. They should also be able to set their own schedule and make adjustments as needed, as long as it is mutually agreed upon. It is important for both parties to make the most of their time together and try to achieve the goals they set together. <br>
+    Please, find more information in Code of Conduct for Mentors/Mentees.
```
Comment (dricazenck) [inline _data/faq.yml:84]: The same applies here.

File: _data/faq.yml (hunk 2 above)
Comment (j4sum) [reply _data/faq.yml:67]: got it!

File: _data/faq.yml (hunk 3 above)
Comment (j4sum) [reply _data/faq.yml:84]: I kept adding and deleting the href attribute but seems like that is what would work here as suggested.

Comment (dricazenck) [reply _data/faq.yml:84]: That is not the best way for this but it is the only way right now. Please also remember to add `target="_self"` to keep on the same page.

File: _data/faq.yml (hunk 1 above)
Comment (dricazenck) [reply _data/faq.yml:66]: you can try also: 

`<a href="/london/code-of-conduct)" target="_self">Code of Conduct for Mentors</a>.` maybe is better.

**Comment by j4sum** [comment]
I have created a new branch with amended code and will create a new PR on it.

**Pull request closed without merging**